projects/packages/classic-theme-helper/src/class-main.php

```diff
@@ -78,15 +76,15 @@ public static function jetpack_load_theme_tools() {
 	/**
 	 * Load theme compat file if it exists.
 	 */
-	public static function jetpack_load_theme_compat() {
+	public static function classic_theme_helper_load_theme_compat() {
```

Comment:
Although this is the Classic Theme Helper Package and its use might be minimal outside from the monorepo, should we deprecate first since it is a public method?

Reply:
Thanks for all the details. After considering them I agree we should be fine by renaming :+1: